jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/src/test/resources/org/jbpm/test/container/archive/localtransactions/tomcat-context.xml

```diff
@@ -6,10 +6,16 @@
    -->
     <Loader delegate="false"/>
 
-    <Listener className="bitronix.tm.integration.tomcat55.BTMLifecycleListener" />
-
     <Resource name="TransactionSynchronizationRegistry" auth="Container" type="javax.transaction.TransactionSynchronizationRegistry"
-              factory="bitronix.tm.BitronixTransactionSynchronizationRegistryObjectFactory" />
+              factory="org.jboss.narayana.tomcat.jta.TransactionSynchronizationRegistryFactory" />
+
+    <Transaction factory="org.jboss.narayana.tomcat.jta.UserTransactionFactory" />
 
-    <Transaction factory="bitronix.tm.BitronixUserTransactionObjectFactory" />
+    <!-- Database resources -->
+    <Resource name="h2DataSource" uniqueName="h2Datasource" auth="Container" type="org.h2.jdbcx.JdbcDataSource" username="sa"
+              user="sa" password="sa" url="jdbc:h2:mem:test-db;MVCC=TRUE;DB_CLOSE_DELAY=-1" description="H2 Data Source" loginTimeout="0"
+              factory="org.h2.jdbcx.JdbcDataSourceFactory"/>
+    <Resource name="jdbc/jbpm" uniqueName="transactionalDataSource" auth="Container" type="javax.sql.DataSource"
+              username="sa" password="sa" driverClassName="com.arjuna.ats.jdbc.TransactionalDriver"
+              url="jdbc:arjuna:java:comp/env/h2DataSource" description="Transactional Driver Data Source"/>
```

Comment:
Why are there 2 datasources?

I suspect this will only work with H2 since I don't see any filtering as previously where there were org.jbpm.datasource.* Maven properties. We need to test with any DBs, not only with H2.